File: python/fizzbuzz.py
fizz = "fizz"
buzz = "buzz"

for x in range(1, 51):
    if(x % 3 == 0 and x % 5 == 0):
        print(fizz + buzz)
    elif(x % 5 == 0):
        print(buzz)
    elif(x % 3 == 0):
        print(fizz)
    else:
        print(x)

File: c#/fizzbuzz.cs
namespace fizzbuzz {
    public class fizzbuzz {
        static void Main(string[] args) {
            for(int i = 1; i <= 50; i++) {
                if(i % 3 == 0 && i % 5 == 0) {
                    System.Console.WriteLine(i + " FizzBuzz");
                } else if(i % 3 == 0) {
                    System.Console.WriteLine(i + " Fizz");
                } else if(i % 5 == 0) {
                    System.Console.WriteLine(i + " Buzz");
                } else {
                    System.Console.WriteLine(i);
                }
            }
        }
    }
}